routes/schedulesRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+const schedulesController = require('../controller/schedulesController');
+const auth = require('../middleware/auth');
+
+//auth.hostVerifyToken 다 집어 넣어야함
+router.get('/:classid',auth.hostVerifyToken,schedulesController.getSchedulesByclassId)
+router.post('/:classid',auth.hostVerifyToken,schedulesController.createSchedule);
+router.put('/update/:scheduleid',auth.hostVerifyToken,schedulesController.updateSchedule);
```

Comment:
```suggestion
router.put('/:scheduleid',auth.hostVerifyToken,schedulesController.updateSchedule);
```
`PUT` 메소드만으로도 수정의 의미를 지닐 수 있기 때문에, RESTful API를 생성하는 기준 관점에서 uri에는 update라는 것을 빼는 것이 좋겠습니다.